internal/testrunner/runners/system/runner.go

```diff
@@ -180,25 +180,25 @@ func (r *runner) run() ([]testrunner.TestResult, error) {
 	}
 
 	// Step 2. Configure package (single data stream) via Ingest Manager APIs.
-	im, err := ingestmanager.NewClient(r.stackSettings.kibana.host, r.stackSettings.elasticsearch.username, r.stackSettings.elasticsearch.password)
```

Comment:
I admit that I didn't analyze the source deeper, but are these Kibana/ES properties still required in stack settings.

Reply:
Good catch, these are not required any more. Removed in 32fe1b2.